tests/acceptance/CreateCardCest.php

```diff
@@ -1,19 +1,13 @@
 <?php
 namespace App\Tests;
 use App\Tests\AcceptanceTester;
-class FirstCest
+class CreateCardCest
```

Comment:
Remove Acceptance folder and any configuration for it

Reply:
done